stories/Header/CompleteHeader/basic.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from 'react';
+
+import { select } from '@storybook/addon-knobs/react'
+import { Button, Collapse, DropdownToggle, DropdownMenu, DropdownItem, Header, Headers, HeaderContent, HeaderSearch, HeaderRightZone, HeaderSocialsZone, HeaderToggler, HeaderLinkZone, Icon, Nav, NavItem, NavLink, HeaderBrand, UncontrolledDropdown } from '../../../src';
+
+// eslint rule has to be disable for few lines here as Storybook addons will go in error
+// if PropTypes are declared in these components
+export class SlimHeader extends Component {
+
+    state = {
+        collapsed: false
+    }
+
+    render() {
+        return (
+            // eslint-disable-next-line react/prop-types
+            <Header type="slim" theme={this.props.theme}>
+                <HeaderContent>
+                    <HeaderBrand>Ente appartenenza/Owner</HeaderBrand>
+                    <HeaderLinkZone>
+                        <HeaderToggler onClick={() => this.setState({ collapsed: !this.state.collapsed })}>
+                            <span>Ente appartenenza/Owner</span>
+                            <Icon icon="it-expand" />
```

Comment:
The cursor behaves like it's an input field. May be due to `href` or type in `HeaderLinkZone. I don't think it should change the cursor to `|`.